### PR TITLE
Initial ticket sales announcement

### PR DESCRIPTION
**Pull request by ericholscher**
This is meant to ship out on Monday, May 25, but we might push it back a day because of the US holiday.

### CODE REVIEW

File: docs/conf/portland/2020/news/online-ticket-sales.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+.. post:: May 26, 2020
+   :tags: portland-2020, tickets
+
+
+Tickets on sale for the Portland virtual conference
+===================================================
+
+We're excited to announce that tickets are on sale for our Portland virtual conference.
+More details will be coming soon,
+but we're getting excited for the accessibility of the new format,
```
Comment (mxsasha) [inline docs/conf/portland/2020/news/online-ticket-sales.rst]:
```suggestion
but we're enthousiastic for the accessibility of the new format,
```
Double use of excited

File: docs/conf/portland/2020/news/online-ticket-sales.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+and can't wait to try out a few new ideas this year.
+
+Get your ticket for the conference
+----------------------------------
+
+**The Portland conference will be online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/portland/2020/>`_ is live, with some preliminary details, and we'll be filling in more as it gets closer.
+
+If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/portland/2020/tickets/>`_! We aren't sure exactly what capacity we will be able to hold for the online event, but we might still sell out to make sure we're providing a managable event side for our attendees, as always.
```
Comment (mxsasha) [inline docs/conf/portland/2020/news/online-ticket-sales.rst]: event side? I don't know what that means.

File: docs/conf/portland/2020/news/online-ticket-sales.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+We're still working out all the details for our online event,
+but know that we're going to keep as much of the spirit of the event alive:
```
Comment (mxsasha) [inline docs/conf/portland/2020/news/online-ticket-sales.rst]:
```suggestion
but know that we're going to keep as much as possible of the spirit of the event alive:
```

File: docs/conf/portland/2020/news/online-ticket-sales.rst (hunk 2 above)
Comment (mxsasha) [inline docs/conf/portland/2020/news/online-ticket-sales.rst]:
```suggestion
If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/portland/2020/tickets/>`_! We aren't sure exactly what capacity we will be able to hold for the online event, but we might still sell out to make sure we're providing a managable event size for our attendees, as always.
```

File: docs/conf/portland/2020/news/online-ticket-sales.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2020/news/online-ticket-sales.rst]:
```suggestion
but we're enthusiastic about the accessibility benefits of the new format,
```

File: docs/conf/portland/2020/lightning-talks.rst [hunk 4]
```diff
@@ -3,6 +3,82 @@
 Lightning Talks
 ===============
 
-.. _speaker-lightning-talks-2020/eu:
+.. _speaker-lightning-talks-2020/na:
```
Comment (thatdocslady) [inline docs/conf/portland/2020/lightning-talks.rst:6]: Any specific reason this is hard-coded into the topic? I thought we specifically moved this type of content to include files..

File: docs/conf/portland/2020/news/online-ticket-sales.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
+:template: {{year}}/generic.html
+
+.. post:: May 26, 2020
+   :tags: portland-2020, tickets
+
+
+Tickets on sale for the Portland virtual conference
```
Comment (thatdocslady) [inline docs/conf/portland/2020/news/online-ticket-sales.rst]:
```suggestion
Announcing ticket sales for the Portland virtual conference
```

File: docs/conf/portland/2020/news/online-ticket-sales.rst [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+* A virtual hallway track
+* A social event of some kind, bring your own beverage and snacks :)
+  
+We really want to make sure that folks can get value from being at the event at the same time,
+and we think it will be another great year of helping our community connect.
+
+Grant program
+-------------
+
+We have updated our grant program application to take into account the online format of the event.
```
Comment (thatdocslady) [inline docs/conf/portland/2020/news/online-ticket-sales.rst]:
```suggestion
We updated our grant program application to take into account the online format of the event.
```

**Review by thatdocslady — COMMENTED** [review]
Looks good, a few nits :)

File: docs/conf/portland/2020/lightning-talks.rst (hunk 4 above)
Comment (ericholscher) [reply docs/conf/portland/2020/lightning-talks.rst:6]: Yea, I don't quite understand why this was pulled in? Guessing there was some implied in-person content?

Comment (plaindocs) [reply docs/conf/portland/2020/lightning-talks.rst:6]: Yup there was. 

A better way might have been to make an online include, but I didn't really have the time and patience to mess around with it.

Comment (plaindocs) [reply docs/conf/portland/2020/lightning-talks.rst:6]: Apologies for not leaving a better commit message making that clear.

**Pull request merged**